controllers/articles-controller.js

```diff
@@ -0,0 +1,11 @@
+const { specificArticles } = require("../models/articles-model")
+
+exports.getArticle = (req, res, next) => {
+    specificArticles(req)
```

Comment:
i would rename this to select article - not a plural so that its clear what this function is doing